Source/CoreExtensions/Public/Core/CoreExtFilePrinterObject.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <CoreMinimal.h>
+#include <UObject/Object.h>
+
+#include "CoreExtFilePrinterObject.generated.h"
+
+UENUM( BlueprintType )
+enum ECoreExtCreateFilePolicy
+{
+    Append,
+    Clear,
+    CreateNew
+};
+
+UCLASS( BlueprintType )
+class COREEXTENSIONS_API UCoreExtFilePrinterObject : public UObject
+{
+    GENERATED_BODY()
+
+public:
+    UCoreExtFilePrinterObject();
+
+    UFUNCTION( BlueprintCallable, meta = ( ReturnDisplayName = "Success" ) )
+    static bool CreateFile( UCoreExtFilePrinterObject *& file_printer, FString file_name, const FString sub_directory, ECoreExtCreateFilePolicy creation_policy );
```

Comment:
all the strings should be passed by const ref in the function parameters

Reply:
Done.